<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="14196" started-at="2017-11-13T04:31:25Z" finished-at="2017-11-13T04:31:39Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="14196" started-at="2017-11-13T04:31:25Z" finished-at="2017-11-13T04:31:39Z">
      <class name="WebAppTest.TestChromeOnGrid">
        <test-method status="PASS" signature="test1()[pri:0, instance:WebAppTest.TestChromeOnGrid@1324409e]" name="test1" duration-ms="4999" started-at="2017-11-12T23:31:34Z" finished-at="2017-11-12T23:31:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
      </class> <!-- WebAppTest.TestChromeOnGrid -->
      <class name="WebAppTest.HeadlessBrowserTest">
        <test-method status="PASS" signature="verifySimplWebAppTitle()[pri:0, instance:WebAppTest.HeadlessBrowserTest@783e6358]" name="verifySimplWebAppTitle" duration-ms="1365" started-at="2017-11-12T23:31:25Z" finished-at="2017-11-12T23:31:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySimplWebAppTitle -->
        <test-method status="FAIL" signature="verifySimplWebAppTitleFail()[pri:0, instance:WebAppTest.HeadlessBrowserTest@783e6358]" name="verifySimplWebAppTitleFail" duration-ms="75" started-at="2017-11-12T23:31:27Z" finished-at="2017-11-12T23:31:27Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at WebAppTest.HeadlessBrowserTest.verifySimplWebAppTitleFail(HeadlessBrowserTest.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySimplWebAppTitleFail -->
      </class> <!-- WebAppTest.HeadlessBrowserTest -->
      <class name="WebAppTest.TestFFonGrid">
        <test-method status="PASS" signature="test1()[pri:0, instance:WebAppTest.TestFFonGrid@180bc464]" name="test1" duration-ms="7724" started-at="2017-11-12T23:31:27Z" finished-at="2017-11-12T23:31:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
      </class> <!-- WebAppTest.TestFFonGrid -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
